From 4e1a8790b04c9930254b64b32df23fd00d59dcdb Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 25 May 2021 13:56:51 +0300
Subject: [PATCH] [SW-43147] i2cslave: support module tdp level command

Support i2c slave direct command '102' which returns
the module static TDP level.

Change-Id: I65833bf2fd6d5299b1b62ac721b0d4f822aa48e8
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/preboot/module_i2c_slave.c b/preboot/module_i2c_slave.c
index 1d420ff..e1bbf43 100644
--- a/preboot/module_i2c_slave.c
+++ b/preboot/module_i2c_slave.c
@@ -124,6 +124,9 @@
 	case HL_MODULE_I2C_FREQ:
 		ret = HL_SLAVE_FREQ_400KHZ;
 		break;
+	case HL_MODULE_TDP_LEVEL:
+		ret = HL_MODULE_TDP_LEVEL_1;
+		break;
 	default:
 		if (HL_I2C_EVENT_IS_PEC(cmd))
 			ret = i2cslave_pec;
